### ISSUE TITLE
Currently view return error 500

### ISSUE THREAD

**Issue by furiml** [shinken-monitoring/mod-webui#529]
Hi everyone,
I'm getting an internal server error when I try to switch to the "currently" view. Here's the traceback : 

```
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/dist-packages/bottle.py", line 862, in _handle
    return route.call(**args)
  File "/usr/local/lib/python2.7/dist-packages/bottle.py", line 1732, in wrapper
    rv = callback(*a, **ka)
  File "/var/lib/shinken/modules/webui2/module.py", line 521, in lock_version
    return f(**args)
  File "/usr/local/lib/python2.7/dist-packages/bottle.py", line 3633, in wrapper
    return template(tpl_name, **tplvars)
  File "/usr/local/lib/python2.7/dist-packages/bottle.py", line 3609, in template
    return TEMPLATES[tplid].render(kwargs)
  File "/usr/local/lib/python2.7/dist-packages/bottle.py", line 3399, in render
    self.execute(stdout, env)
  File "/usr/local/lib/python2.7/dist-packages/bottle.py", line 3386, in execute
    eval(self.co, env)
  File "/var/lib/shinken/modules/webui2/plugins/dashboard/views/currently.tpl", line 625, in <module>
    %pct_known_problems=round(100.0 * known_problems / s['nb_elts'], 2)
ZeroDivisionError: float division by zero
```

Looking at previous topics, I first though this was in the shinken/misc/datamanager.py file, because atm I don't have any service declared. But looking at the traceback it seems it comes from the currently.tpl file and I must say I have no idea what kind of file this is !_

Can you help ?
Thanks !

**Comment by mohierf** [comment]
This error occurs because you did not defined any service ... as of it `s['nb_elts']` (which contains the number of services...) is 0! This should be catched to avoid such an error ... but a configuration with no services is not common... I would say that it should not even exists :wink: 

**fyi**: tpl files are html templates that contain some python code to build HTML pages :wink:

**Comment by kp01aj** [comment]
Hi,

I had the same problem.

I solved as follows:

Comment out the validation in the file

`/var/lib/shinken/modules/webui2/plugins/dashboard/views/currently.tpl
`

**On line 626**

`% Pct_known_problems = round (100.0 * known_problems / s [ 'nb_elts'], 2)
`
**Was as follows:**

`<-!% Pct_known_problems = round (100.0 * known_problems / s [ 'nb_elts'], 2) ->`

Temporary solution until a new version.

**Comment by maethor** [comment]
This is fixed with commit ccdf3134 that will be in the next version.

Thanks.
